mlir/test/Dialect/Tosa/canonicalize.mlir

```diff
@@ -613,3 +613,16 @@ func.func nested @fold_tile_rank_zero() -> tensor<i32> {
   %1 = tosa.tile %0 {multiples = array<i64>} : (tensor<i32>) -> tensor<i32>
   return %1 : tensor<i32>
 }
+
+// -----
+
+// CHECK-LABEL: @fold_reciprocal
+func.func nested @fold_reciprocal() -> tensor<3x600x1200xf32> {
```

Comment:
We may want to add the test case to check the int-type operation is not folded?

Reply:
Ack, added.

Reply:
In the spec, TOSA doesn't support reciprocal on integer types at all, but this isn't enforced in the dialect as it allows any Tensor type.

Reply:
Good to know (Currently even boolean can be given), need to update this Operation.